cli/dts/README.md

```diff
@@ -4,16 +4,26 @@ The files in this directory are mostly from the TypeScript repository. We
 currently (unfortunately) have a rather manual process for upgrading TypeScript.
 It works like this currently:
 
-1. Checkout typescript repo in a separate directory.
-2. Copy typescript.js into Deno repo.
-3. Copy d.ts files into dts directory.
+1. Checkout denoland/typescript repo in a separate directory.
+1. Add Microsoft/TypeScript as a remote and fetch its latest tags
+1. Checkout a new branch based on this tag.
+1. Cherry pick the custom commit we made in a previous release to the new one.
+1. This commit has a "deno.ts" file in it. Read the instructions in it.
```

Comment:
Can we be more specific here? Maybe add a branch `floating-patches` that should be rebased on top of the latest release branch?

Reply:
Do you mean about this point? I think it's more ideal to keep those instructions closest to that code.

For an additional branch, I think we should have a single source of truth, which is the previous release branch.

Reply:
Okay I guess that makes sense, I retract that comment.